Changes

```diff
@@ -32,6 +32,9 @@ Working version
   unbound Unix socket. Add support for receiving abstract (Linux) socket paths.
   (Tim Cuthbertson, review by Sébastien Hinderer and Jérémie Dimino)
 
+- GRP#2263: Delete the deprecated Bigarray.*.map_file functions
```

Comment:
```suggestion
- GPR#2263: Delete the deprecated Bigarray.*.map_file functions in favour of `*_of_genarray (Unix.map_file ...)` functions instead. The `Unix.map_file` function was introduced in OCaml 4.06.0 onwards.
```

And just to be clear, is there a compat story for pre-4.06 via a shim library? If so, it would be helpful to mention that in the changelog also. Otherwise it looks like we'll be revving a number of MirageOS libraries to a minimum of OCaml 4.06 quite shortly :-)

Reply:
Would that be a bad thing? :)

Reply:
I don't know if it's already written, but I can't see a technical problem with a shim for < 4.06.0 which updates the `Unix` signature. Of course, that means that all the MirageOS code will need updating to use the *new* library, and activate the shim in old code, right?